tests/baselines/reference/unknownType1.errors.txt

```diff
@@ -221,6 +222,8 @@ tests/cases/conformance/types/unknown/unknownType1.ts(181,5): error TS2322: Type
     // Functions with unknown return type don't need return expressions
     
     function f27(): unknown {

```

Comment:
Just to be sure: This error should be emitted because `noImplicitReturns` is `true` by default right?

Reply:
Got it.

Reply:
Wait then [this comment](https://github.com/microsoft/TypeScript/blob/b6fed21a872feb898971abba009bf4171d9ec871/tests/cases/conformance/types/unknown/unknownType1.ts#L150) is incorrect.

Reply:
I'm so confused. Can we get a playground link of what was desirable in TS 5.0, but is now incorrect in nightly?